bundle/libraries/libraries.go

```diff
@@ -0,0 +1,43 @@
+package libraries
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/libs/log"
+)
+
+type attach struct {
+}
+
+func AttachToResources() bundle.Mutator {
+	return &attach{}
+}
+
+func (a *attach) Name() string {
+	return "libraries.AttachToResources"
+}
+
+func (a *attach) Apply(ctx context.Context, b *bundle.Bundle) error {
+	r := b.Config.Resources
+	for k := range b.Config.Resources.Jobs {
+		tasks := r.Jobs[k].JobSettings.Tasks
+		for tK := range tasks {
+			task := tasks[tK]
```

Comment:
tK is an int? If so `i` would be better IMO.

This can be `task := &tasks[i]` to make code below less verbose when mutating it.